x509-identity-mgmt/js/src/routes/trusted-cas-router.js

```diff
@@ -0,0 +1,50 @@
+
+const express = require('express');
+
+const HttpStatus = require('http-status-codes');
+
+const service = require('../services/trusted-cas-service');
+
+const { validateNewTrustedCA, validateUpdTrustedCA } = require('../core/schema-validator');
+
+const router = express.Router();
+
+router.route('/ca')
+  /* Root CA Certificate */
+  .get(async (req, res) => {
+    const result = await service.getRootCertificate();
+    res.status(HttpStatus.OK).json(result);
+  });
+
+router.route('/ca/crl')
+  /* Latest CRL issued by the Root CA */
+  .get(async (req, res) => {
+    const result = await service.getRootCRL();
+    res.status(HttpStatus.OK).json(result);
+  });
+
+router.route('/trusted-cas')
+  /* Register Trusted CA Certificate */
+  .post(validateNewTrustedCA(), (req, res) => {
+    res.sendStatus(201);
```

Comment:
Is it possible to use a `HttpStatus` constant instead of a hardcoded number ?

Reply:
Yes, these routes are not yet implemented, but I will do that, this will be the next feature.

Reply:
Ok, I got it.